modules/workergroup/variables.tf

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2022, 2023 Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
+
+variable "enabled" {
```

Comment:
What is this enabling?

Reply:
Clarified description. Internal sub-module vars are kept short for cleaner definition on worker_pools, since defaults are merged with pool definitions and keys should match.